weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -293,34 +294,58 @@ def otel_export(self, req: tsi.OtelExportReq) -> tsi.OtelExportRes:
             ),
         ).objs
 
+        # FILE BYPASS START
+        if len(existing_objects) == 0:
+            # We don't know if any existing ops have been created
```

Comment:
Is this comment correct? Isn't it more like
```suggestion
            # None of the ops already exist, we create one, same for all, since no code capture available for OTel
```
?